packages/manager/modules/pci/src/projects/new/payment/components/default/controller.js

```diff
@@ -6,4 +8,12 @@ export default class PciProjectNewPaymentDefaultCtrl {
       myAccount: { application: 'dedicated', path: '#/useraccount/dashboard' },
     });
   }
+
+  getFormatCreditCardNumber() {
+    return `····${this.defaultPaymentMethod.label.replace(/\D/g, '')}`;
```

Comment:
Hmm should you not get only the last credit card number here?

Reply:
Ah good point. 

Done